lib/taste_tester/tunnel.rb

```diff
@@ -54,9 +57,7 @@ def cmd
       # In most cases the first request from chef was "breaking" the tunnel,
       # in a way that port was still open, but subsequent requests were hanging.
       # This is reproducible and should be looked into.
-      cmd = "#{ssh_base_cmd} -o ServerAliveInterval=10 " +
-        "-o ServerAliveCountMax=6 -f -R #{@port}:localhost:#{@server.port} "
-      build_ssh_cmd(cmd, [cmds])
+      build_ssh_cmd(ssh_base_cmd, [cmds])
```

Comment:
doesn't this need `@extra_options` ?

Reply:
The way I have set this up is to assign `@extra_options` at initialize and then just use it in `ssh_vanilla_cmd` in `SSH::Util`. Let me know if that works.

Reply:
Oh I missed that, nice.